src/db/mod.rs

```diff
@@ -169,15 +175,14 @@ impl ThreadedDatabase for Database {
             read_preference: Option<ReadPreference>,
             write_concern: Option<WriteConcern>)
             -> Database {
-        let rp = read_preference.unwrap_or_else(|| client.read_preference.to_owned());
-        let wc = write_concern.unwrap_or_else(|| client.write_concern.to_owned());
-
         Arc::new(DatabaseInner {
-            name: String::from(name),
-            client: client,
-            read_preference: rp,
-            write_concern: wc,
-        })
+                     name: String::from(name),
```

Comment:
Because I removed the read preference and write concern options from `Client`, I changed the defaults for them on `Database` to be the previous defaults that `Client` had.